src/components/App/App.jsx

```diff
@@ -150,7 +150,12 @@ const App = (props) => {
         <Helmet
           defaultTitle={strings.title_default}
           titleTemplate={strings.title_template}
-        />
+        >
```

Comment:
isn't this the top?

Reply:
Yes, we have to add a script to the header and to where we want it to render.